sgxs-loaders/src/win_sgx/mod.rs

```diff
@@ -0,0 +1,342 @@
+use winapi::um::memoryapi::VirtualFree;
+use generic::{self, EinittokenError, EnclaveLoad, Mapping};
+use std::io::{Error as IoError, Result as IoResult};
+use winapi::um::processthreadsapi::GetCurrentProcess;
+use winapi::um::winnt::{HANDLE, ENCLAVE_INIT_INFO_SGX, MEM_RELEASE};
+use winapi::um::enclaveapi::{CreateEnclave, InitializeEnclave, IsEnclaveTypeSupported, LoadEnclaveData};
+use winapi::_core::ffi::c_void;
+use abi::{Attributes, Einittoken, Miscselect, PageType, SecinfoFlags, Secs, Sigstruct};
+use sgxs_crate::sgxs::{MeasEAdd, MeasECreate, PageChunks, SgxsRead};
+use sgxs_crate::einittoken::EinittokenProvider;
+use sgxs_crate::loader;
+use std::fs::OpenOptions;
+
+use std::{mem, ptr};
+mod defs;
+use self::defs::*;
+use std::sync::Arc;
+use crate::{MappingInfo, Tcs};
+use std::path::Path;
```

Comment:
Order imports as follows:
1st block: Anything from standard Rust crates (`std`, `core`, `proc_macro`, etc.)
2nd block: Anything from 3rd-party dependencies
3rd block: Anything from our dependencies
4th block: Anything from this crate